extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -400,6 +400,15 @@ export class ProjectsController {
 		return project.files.find(x => utils.getPlatformSafeFileEntryPath(x.relativePath) === utils.getPlatformSafeFileEntryPath(utils.trimUri(context.root.uri, context.uri)));
 	}
 
+	/**
+	 * Opens the folder containing the project
+	 * @param context a treeItem in a project's hierarchy, to be used to obtain a Project
+	 */
+	public async openContainingFolder(context: BaseProjectTreeItem): Promise<void> {
```

Comment:
Is there a way to add some test around it? Just to make sure that the functionality is correct.

Reply:
I don't think a unit test would help much since the folder opening would be mocked :/ This is something an integration test would be better for once we have those set up

Reply:
You have access to the filesystem so creating and opening folders is perfectly doable (we do this for the book tests)

Reply:
discussed with Charles and determined adding a test for this is unnecessary since it would just be testing that the vs code api works, which we can safely assume is the case most of the time.